Status[] behaviours
Status[] goals
float32  activationThreshold
float32  activationThresholdDecay
string[] runningBehaviours
string[] influencedSensors
int32    stepCounter
string[] plan
uint32   plan_index